### PR TITLE
fix: update Project governance page to match reality

### PR DESCRIPTION
**Pull request by joyeecheung**
I noticed that the governance page (source is https://github.com/nodejs/nodejs.org/blob/main/pages/en/about/governance.md) appears to be out of sync and probably is copied from an older document that used incorrect descriptions. At least it does not match what we say in https://github.com/nodejs/node/blob/main/GOVERNANCE.md now and I remember we made sure that it was clear it's collaborators who nominate collaborators (which is also what we do in practice).

Here is my attempt at clarifying the text.

cc @nodejs/tsc

### Check List

- [ ] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing.
- [ ] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by GeoffreyBooth — APPROVED** [review]
Should this just _be_ the other governance doc, rather than being a separate file that can get out of sync?

**Review by AugustinMauroy — APPROVED** [review]
LGTM !

**Review by targos — APPROVED** [review]

File: pages/en/about/governance.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ The Node.js project follows a [Consensus Seeking][] decision making model.
 ## Collaborators
 
 The [nodejs/node][] core GitHub repository is maintained by the Collaborators
-who are added by the Technical Steering Committee ([TSC][]) on an ongoing basis.
+who are nominated by other existing Collaborators on an ongoing basis.
```
Comment (mcollina) [inline pages/en/about/governance.md:15]: That's not what our charter says. Ref https://github.com/nodejs/TSC/blob/main/TSC-Charter.md#section-4-responsibilities-of-the-tsc

**Review by mcollina — CHANGES_REQUESTED** [review]

File: pages/en/about/governance.md (hunk 1 above)
Comment (aduh95) [reply pages/en/about/governance.md:15]: TSC voting members, as org admin, are the ones who technically add the GH account to the GH team. But definitely the nomination and approval is made by existing collaborators. I prefer Matteo’s suggestion.

**Comment by ovflowd** [comment]
> Should this just _be_ the other governance doc, rather than being a separate file that can get out of sync?

You mean as a link to the actual Markdown doc? I'm fine with that, also.

**Comment by GeoffreyBooth** [comment]
> You mean as a link to the actual Markdown doc?

However you want to do it; a link, or pulling that document into the build for the website somehow, whatever. Having two copies of it just means two things that can get out of sync, or worse, two places to haggle over language.

**Comment by joyeecheung** [comment]
I think it's nice to have a brief summary of the governance model - I doubt how many browsing our websites would be entertained reading the long governance doc. Another solution is to have a TL;DR in the governance doc, and somehow sync that part to the website, or on the website only render the TL;DR

File: pages/en/about/governance.md (hunk 1 above)
Comment (joyeecheung) [reply pages/en/about/governance.md:15]: I don't think the charter says anything about nomination? The only relevant part is:

> the TSC voting members are responsible for all technical development within the Node.js project, including:...
> Maintaining the list of additional Collaborators.

Which isn't contradicting with the descriptions here. The TSC is responsible of maintaining the list, but collaborators can also propose changes to that list. When there are disagreements the TSC is responsible of finding a resolution, that's all.

Also to reference our governance doc https://github.com/nodejs/node/blob/main/GOVERNANCE.md#collaborator-nominations:

> Existing collaborators can nominate someone to become a collaborator.

**Review by aymen94 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**